### DISCUSSION TITLE
Remapping dependencies

### DISCUSSION THREAD

**Discussion by tsvilans** [visose/Robots#75] [Q&A] [answered]
Moving towards using NuGet for RhinoCommon and Grasshopper dependencies would make it easier to keep dependencies consistently targeted and updated: http://developer.rhino3d.com/guides/rhinocommon/using-nuget/

**Comment by visose** [comment]
Yes, great idea. I was thinking of target it now to Rhino 6 now that it's officially released. Using Nuget will be the best way to go for this. The package descriptions still say Rhino WIP but i guess they haven't updated the description yet, they look very new.

**Comment by tsvilans** [comment]
From chats on Discourse, and in the discussions about Speckle, it seems McNeel encourages it.

**Comment by visose** [comment, marked as answer]
Closing this as resolved in the current codebase. The projects now use SDK-style project files with NuGet package references for RhinoCommon, Grasshopper, and Rhino3dm where appropriate, so the dependency-remapping request is no longer an open action. Please open a new discussion if there is a current package/dependency issue.
